features/place/read.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Read places
+
+  Background:
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I send and accept "application/json"
+
+  @bugfix # https://jira.uitdatabank.be/browse/III-5979
+  Scenario: Try to get a event that actually is a place
+    Given I create a minimal place and save the "url" as "placeUrl"
+    # The line below gets an event which is actually a place - this is on purpose and should fail
```

Comment:
When using a step like `iGetTheEventAtAndTheResponseIs` the comment is not needed. Furthermore the test is marked with a bug ticket so all is clear from the ticket.